config/initializers/00-load_config.rb

```diff
@@ -16,6 +16,7 @@
   ENV['CRANTASTIC_PASSWORD'] = private_config["CRANTASTIC_PASSWORD"]
   ENV['RECAPTCHA_PRIVATE_KEY'] = private_config["RECAPTCHA_PRIVATE_KEY"]
   ENV['RECAPTCHA_PUBLIC_KEY'] = private_config["RECAPTCHA_PUBLIC_KEY"]
+  ENV['SECRET_TOKEN'] = private_config["SECRET_TOKEN"]
   ENV['TWITTER_OAUTH_TOKEN_SECRET'] = private_config["TWITTER_OAUTH_TOKEN_SECRET"]
   ENV['TWITTER_OAUTH_TOKEN'] = private_config["TWITTER_OAUTH_TOKEN"]
   ENV['TWITTER_CONSUMER_KEY'] = private_config["TWITTER_CONSUMER_KEY"]
```

Comment:
Lets replace this file with [Figaro](https://github.com/laserlemon/figaro). Add config/application.yml, add it to gitignore, and move all this to that file (and this private_config stuff)